src/test/scala/org/zalando/spearheads/innkeeper/services/RoutesServiceSpec.scala

```diff
@@ -325,7 +325,9 @@ class RoutesServiceSpec extends FunSpec with Matchers with MockFactory with Scal
     ownedByTeam = ownedByTeam,
     createdBy = createdBy,
     createdAt = createdAt,
-    updatedAt = updatedAt)
+    updatedAt = updatedAt,
+    hasStar = false
```

Comment:
I consider the collision detection tests and the serializing to eskip test as sufficient for `hasStar = true`. Those cover the business logic related to hasStar.